clang/tools/3c/utils/port_tools/generate_ccommands.py

```diff
@@ -190,7 +190,12 @@ def run3C(checkedc_bin, compile_commands_json, checkedc_include_dir, skip_paths,
     args = []
     args.append(prog_name)
     args.extend(DEFAULT_ARGS)
-    args.extend(list(set(total_x_args)))
+    args.extend(extra_3c_args)
+    args.append('-p')
+    args.append(compile_commands_json)
+    # Same as in getCheckedCArgs
+    args.append('-extra-arg-before=-I' + checkedc_include_dir)
+    args.append('-extra-arg-before=-w')
```

Comment:
This seems OK to me, but I'm not intimately familiar with the details. Perhaps @kyleheadley or @john-h-kastner should quickly sanity-check.